data/hooks/conf_regen/15-nginx

```diff
@@ -49,7 +49,14 @@ do_pre_regen() {
                             | jq ".certificates.\"$domain\".CA_type" \
                             | tr -d '"')
 
+    if [[ -e "/etc/ssl/private/dh2048.pem" ]]; then
+      export dh_params=true
+    else
+      export dh_params=false
+    fi                  
+
     ynh_render_template "server.tpl.conf" "${nginx_conf_dir}/${domain}.conf"
+    ynh_render_template "plain/yunohost_admin.conf" "${nginx_conf_dir}/yunohost-admin.conf"
```

Comment:
If the template is to be customized, I think it shouldn't be in "plain" anymore

Reply:
mh... ok. So in the same directory as server.tpl.conf ?

Reply:
Yup

Reply:
done